backend.native/tests/framework/stdlib/stdlib.swift

```diff
@@ -216,10 +216,10 @@ class StdlibTests : TestProvider {
         try applyVoid { $0.add("bar") }
         try applyVoid { $0.remove("baz") }
         try applyVoid { $0.add("baz") }
-        try applyVoid { $0.add(TripleVals(first: 1, second: 2, third: 3)) }
-        try apply { $0.member(TripleVals(first: 1, second: 2, third: 3)) as! TripleVals }
+        try applyVoid { $0.add(TripleVals<NSNumber>(first: 1, second: 2, third: 3)) }
```

Comment:
Just curious, why it is needed to specify `NSNumber`?

Reply:
Swift compiler infers `Int` as type argument, but Objective-C classes can have only "class" (e.g. reference) types as type arguments.